contracts/ethereum/context-proxy/test/ContextProxy.t.sol

```diff
@@ -96,6 +96,67 @@ contract ContextProxyTest is Test {
         mockExternal = new MockExternalContract();
     }
 
+    function testExecuteWithSingleApproval() public {
+    // First, verify the initial numApprovals is 3 (default)
+    assertEq(proxy.numApprovals(), 3, "Initial numApprovals should be 3");
```

Comment:
but this isn't what we want.. it should be 1 by default, overridable to anything, for example.. 3

Reply:
Okay I'll update it